src/components/Nav.jsx

```diff
@@ -0,0 +1,22 @@
+import { fetchTopics } from "../api";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+export default function Nav() {
+  const [topic, setTopic] = useState([]);
+  useEffect(() => {
+    fetchTopics().then((response) => {
+      console.log(response);
```

Comment:
This isn't needed yet, but make sure to remove the console.log and include some kind of loading state for this once it's fully implemented